setup.py

```diff
@@ -11,7 +11,8 @@
 
 def read_file(filename):
     """Get the contents of a file"""
-    with open(join(abspath(dirname(__file__)), filename)) as file:
+    path = join(abspath(dirname(__file__)), filename)
+    with open(path, encoding='UTF-8') as file:
```

Comment:
Note that this code would be migrated to the more modern `pathlib` by the [changes in PR #86](https://github.com/vshn/concierge-cli/pull/86/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R13). Maybe you want to merge that PR first?

Reply:
Good idea, done.